src/ui/widgets/FlexContainer/flexContainer.tsx

```diff
@@ -11,18 +11,14 @@ const FlexProps = {
 };
 
 export const FlexContainerComponent = (
-  props: InferWidgetProps<typeof FlexProps> & Component
+  props: InferWidgetProps<typeof FlexProps>
 ): JSX.Element => {
   const classNames = [classes.FlexContainer];
   const { flexFlow = null } = props;
   if (flexFlow !== null) {
     classNames.push(classes[flexFlow]);
   }
-  return (
-    <div className={classNames.join(" ")} style={props.style}>
-      {props.children}
-    </div>
-  );
+  return <div className={classNames.join(" ")}>{props.children}</div>;
```

Comment:
I think FlexContainer needs to know how big it should be? Is this still working as expected?